data/autoyast_sle15/pc_tools.xml

```diff
@@ -0,0 +1,159 @@
+            <listentry>
+                <media_url>http://download.suse.de/ibs/Devel:/PubCloud:/CI/SLE_15</media_url>
+                <product>public_cloud_devel</product>
+                <alias>Public Cloud Devel</alias>
+                <product_dir>/</product_dir>
+                <priority config:type="integer">20</priority>
+                <ask_on_error config:type="boolean">false</ask_on_error>
+                <confirm_license config:type="boolean">false</confirm_license>
+                <name>Public Cloud Devel</name>
+                <ask_user config:type="boolean">false</ask_user>
+                <selected config:type="boolean">true</selected>
+            </listentry>
+            <listentry>
+                <media_url>https://download.opensuse.org/repositories/devel:/languages:/python:/backports/SLE_15/</media_url>
+                <product>python_devel</product>
+                <alias>Python Devel</alias>
+                <product_dir>/</product_dir>
+                <priority config:type="integer">20</priority>
+                <ask_on_error config:type="boolean">false</ask_on_error>
+                <confirm_license config:type="boolean">false</confirm_license>
+                <name>Python Devel</name>
+                <ask_user config:type="boolean">false</ask_user>
+                <selected config:type="boolean">true</selected>
+            </listentry>
+        </add_on_products>
+    </add-on>
+    <suse_register>
+        <do_registration config:type="boolean">true</do_registration>
+        <email/>
+        <reg_code>{{SCC_REGCODE}}</reg_code>
+        <install_updates config:type="boolean">true</install_updates>
+        <addons config:type="list">
+            <addon>
+                <name>sle-module-basesystem</name>
+                <version>15</version>
+                <arch>x86_64</arch>
+            </addon>
+            <addon>
+                <name>sle-module-public-cloud</name>
+                <version>15</version>
+                <arch>x86_64</arch>
+            </addon>
+            <addon>
+                <name>sle-module-development-tools</name>
+                <version>15</version>
+                <arch>x86_64</arch>
+            </addon>
+            <addon>
+                <name>sle-module-legacy</name>
+                <version>15</version>
+                <arch>x86_64</arch>
+            </addon>
+        </addons>
+    </suse_register>
+    <bootloader>
+        <device_map config:type="list">
+            <device_map_entry>
+                <firmware>hd0</firmware>
+                <linux>/dev/vda</linux>
+            </device_map_entry>
+        </device_map>
+        <global>
+            <timeout config:type="integer">1</timeout>
+            <append>splash=verbose</append>
+        </global>
+    </bootloader>
+    <networking>
+        <keep_install_network config:type="boolean">true</keep_install_network>
+    </networking>
+    <software>
```

Comment:
I don't see a part for pop-up messages, which you might want to ignore:
See https://www.suse.com/documentation/sles-12/singlehtml/book_autoyast/book_autoyast.html#CreateProfile.Reporting
```xml
    <report>
      <errors>
        <log config:type="boolean">true</log>
        <show config:type="boolean">false</show>
        <timeout config:type="integer">-1</timeout>
      </errors>
      <messages>
        <log config:type="boolean">true</log>
        <show config:type="boolean">false</show>
        <timeout config:type="integer">-1</timeout>
      </messages>
      <warnings>
        <log config:type="boolean">true</log>
        <show config:type="boolean">false</show>
        <timeout config:type="integer">-1</timeout>
      </warnings>
      <yesno_messages>
        <log config:type="boolean">true</log>
        <show config:type="boolean">false</show>
        <timeout config:type="integer">-1</timeout>
      </yesno_messages>
    </report>
```

Reply:
don't think that I want to continue in case of error . better to fail the job to not overwrite last good image with somehow broken one , but everything else make sense to add it

Reply:
Some errors cannot be skipped, even  with the setting. But in case you have some info/warning pop-ups, by default they have 10 seconds timeout, meaning, won't break installation, but just make it longer. So no actual need for that.

Reply:
nice hint